systests/microprofile-tck/pom.xml

```diff
@@ -0,0 +1,137 @@
+<?xml version="1.0"?>
```

Comment:
This seems like its already covered by `/systests/microprofile/client/weld/` ?  I had intended to add client tests for weld and owb at some point, and if CXF implements other specs (talk around OpenAPI support in a JIRA ticket) then it could all live in one tree.

Reply:
Right - I tried to move those tests to another systest tree, but something weird happened with git - instead of showing that they moved, it shows that the files were deleted and then re-created.  I'll try to fix this so that the history is more accurate.  And I will try to refactor it so that they can all live under the same directory tree.  Thanks!